src/plugin/index.ts

```diff
@@ -181,9 +181,9 @@ export const cssModules = (
 				Object.entries(cssModule.exports).map(async ([exportName, exported]) => {
 					if (
 						exportName === 'default'
-						&& exportMode !== 'named'
+						&& exportMode === 'named'
 					) {
-						this.warn('You cannot use "default" as a class name as it conflicts with the default export. Set "exportMode: named" to use "default" as a class name.');
+						this.warn('You cannot use "default" as a class name together with "exportMode: named" as it conflicts with the default export.');
```

Comment:
This is inaccurate. You _can_ use `default` as a classname with `exportMode: named`.

Please revert.